src/DynamoCore/Search/SearchElements/ZeroTouchSearchElement.cs

```diff
@@ -53,6 +54,8 @@ public ZeroTouchSearchElement(FunctionDescriptor functionDescriptor)
             Description = functionDescriptor.Description;
             Assembly = functionDescriptor.Assembly;
 
+            //Create full name including assembly name
+            fullname = string.Format("{0}.{1}", System.IO.Path.GetFileNameWithoutExtension(Assembly), functionDescriptor.QualifiedName);
```

Comment:
Do we not want to use semi-colon as separator here? Right now assembly name part of the `fullname` is indistinguishable from the actual fully qualified name of a method, we may not have immediate need of telling one from the other, but assuming we do need to separate it out for some reason, this dot will make it impossible. Does that make sense?